examples/simple-source/src/main.rs

```diff
@@ -63,6 +65,7 @@ pub(crate) mod simple_source {
                         },
                         event_time: chrono::offset::Utc::now(),
                         keys: vec![],
+                        headers:headers.clone()
                     })
```

Comment:
do we need to clone?

Reply:
Yeah cloning is not good ,let me modify it

Reply:
why can't we just give the ownership? are we using the headers anywhere else?

Reply:
the problem is every for loop iteration will be new scope ,if there was only one iteration it would have worked.

Reply:
missed the for loop.